code/game/objects/items/environmental_regulator.dm

```diff
@@ -20,14 +20,14 @@
 
 /obj/item/environmental_regulator/Destroy()
 	owner = null
-	STOP_PROCESSING(SSprocessing, src)
+	STOP_PROCESSING(SSobj, src)
 	return ..()
 
 /obj/item/environmental_regulator/equipped(mob/user, slot, initial)
 	. = ..()
 	switch(slot)
 		if(ITEM_SLOT_BACK, ITEM_SLOT_SUITSTORE)
-			START_PROCESSING(SSprocessing, src)
+			START_PROCESSING(SSobj, src)
```

Comment:
I reallly would put this on SSmobs and make SSmobs a two-phase subsystem, one for atoms and one for mobs. That way this can NEVER happen again.